workspace/src/operation.rs

```diff
@@ -172,7 +184,7 @@ impl From<workspaces::result::ViewResultDetails> for ViewResultDetails<H256> {
     }
 }
 
-#[cfg(not(feature = "ethabi"))]
+// #[cfg(not(feature = "ethabi"))]
```

Comment:
I'd delete this if it doesn't have any sense.

Reply:
I removed it.